readme.md

```diff
@@ -4,10 +4,13 @@
 
 [![npm status](http://img.shields.io/npm/v/remark-git-contributors.svg)](https://www.npmjs.org/package/remark-git-contributors)
 [![node](https://img.shields.io/node/v/remark-git-contributors.svg)](https://www.npmjs.org/package/remark-git-contributors)
-[![Travis build status](https://img.shields.io/travis/vweevers/remark-git-contributors.svg?label=travis)](http://travis-ci.org/vweevers/remark-git-contributors)
-[![AppVeyor build status](https://img.shields.io/appveyor/ci/vweevers/remark-git-contributors.svg?label=appveyor)](https://ci.appveyor.com/project/vweevers/remark-git-contributors)
-[![Dependency status](https://img.shields.io/david/vweevers/remark-git-contributors.svg)](https://david-dm.org/vweevers/remark-git-contributors)
+[![Travis build status](https://img.shields.io/travis/remarkjs/remark-git-contributors.svg?label=travis)](http://travis-ci.org/remarkjs/remark-git-contributors)
+[![AppVeyor build status](https://img.shields.io/appveyor/ci/remarkjs/remark-git-contributors.svg?label=appveyor)](https://ci.appveyor.com/project/remarkjs/remark-git-contributors)
```

Comment:
This one shows as:

![image](https://user-images.githubusercontent.com/3055345/49303358-d2615c80-f4c9-11e8-96e2-1e3f7698e978.png)

Reply:
Weird. I added it but it’s stuck on https://ci.appveyor.com/project/wooorm/remark-git-contributors now... How does one fix this 🤔

Reply:
It’s there!